service/history/execution/mutable_state_builder.go

```diff
@@ -316,6 +316,8 @@ func (e *mutableStateBuilder) CopyToPersistence() *persistence.WorkflowMutableSt
 	state.BufferedEvents = e.bufferedEvents
 	state.VersionHistories = e.versionHistories
 	state.Checksum = e.checksum
+	state.ReplicationState = e.replicationState
+	state.ExecutionStats = e.executionStats
```

Comment:
did the roundtrip test uncover these missed fields?

Reply:
yes